tony-core/src/test/java/com/linkedin/tony/TestTonyClient.java

```diff
@@ -104,4 +104,16 @@ public void testValidateTonyConfRequestedGpusUnderLimit() {
     conf.setInt(TonyConfigurationKeys.getMaxTotalResourceKey(Constants.GPUS), 10);
     TonyClient.validateTonyConf(conf);
   }
+
+  @Test
+  public void testTonyApplciatonTags() throws Exception {
+    final String applicationTags = "flow_id:1,exec_id:0,proj_name:demo";
+    String[] args = { "-conf",
+        TonyConfigurationKeys.APPLICATION_TAGS + "=" + applicationTags };
+    TonyClient client = new TonyClient();
+    client.init(args);
+    assertEquals(
+        applicationTags,
+        client.getTonyConf().get(TonyConfigurationKeys.APPLICATION_TAGS));
```

Comment:
Nit: based on the Javadoc of TestNG's `assertEquals`, the `actual` argument should come first, then the `expected`. So I would swap lines 116 and 117.

Reply:
Good catch